src/utils/logout.ts

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+export async function keycloackSessionLogOut() {
+  try {
+    await fetch('/api/auth/logout');
+  } catch (error) {
+    throw new Error(`Could not log out from Keycloak`);
```

Comment:
can we also add original error here, otherwise we are masking it and if it happens, will be harder to debug

Reply:
My concern is that this function will be called in the browser and consequently we would log some internal information in the browser. I can add a log in the API route though, it will be displayed in the server

Reply:
not sure what could be logged there, unless keycloak logs some internal stuff, which would make it worst idp ever

Reply:
but it's fine for me, I would not expect that we have issues there often or at all 😄

Reply:
True it should disclose sensitive stuff normally 😄

I put the log on the server side just in case